package.json

```diff
@@ -79,5 +81,18 @@
     "*.js": [
       "eslint --format=codeframe --rulesdir='./scripts/eslint_rules'"
     ]
+  },
+  "jest": {
```

Comment:
wouldn't we need to alias `packages/*` to their respective `src` directories to have a great watch mode?

Reply:
I think I'm missing something, because I'm not sure that I understood that

Reply:
Sorry if I was not clear. The source code in packages gets transpiled in a build step, to have great save+watch tests experience I think it would be better to integrate transpiling into jest, rather than run separate build with watch mode.

Because packages are linked and treated as regular npm dependencies, the regular node's dependency algorithm kicks in and jest will search for the built files rather than source ones.